packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -732,8 +732,8 @@ export default function useAutocomplete(props) {
       'aria-autocomplete': autoComplete ? 'both' : 'list',
       'aria-controls': popupOpen ? `${id}-popup` : null,
       // Disable browser's suggestion that might overlap with the popup.
-      // (autocomplete and autofill)
-      autoComplete: 'disabled',
+      // Handle autocomplete but not autofill.
+      autoComplete: 'off',
```

Comment:
It also fixes HTML5 validation errors: https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-18376--material-ui.netlify.com%2Fcomponents%2Fautocomplete%2F